src/mscorlib/shared/System/Runtime/CompilerServices/IntrinsicAttribute.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor, Inherited = false)]
```

Comment:
This might (?) be a good place for a comment indicating that the jit MAY replace direct calls to `[Intrinsic]` methods with its own sequence, but that it also may opt to fall back on the MSIL implementation (and presumably indirect calls will execute the fall-back implementation?).

Reply:
Added.